pkg/knuu/knuu.go

```diff
@@ -49,8 +49,6 @@ type Options struct {
 }
 
 func New(ctx context.Context, opts Options) (*Knuu, error) {
-	opts.TestScope = k8s.SanitizeName(opts.TestScope)
```

Comment:
Removing this will break the app tests/simulations again.

The app repo still uses the deprecated version to initialize knuu. This results in a mismatch of the scope and the namespace.

The namespace is sanitized but not the scope.

Reply:
Also the scope is sanitized in the `setDefault` func here: https://github.com/celestiaorg/knuu/blob/07440a77e606ab29976d42c050f2b47c08dd87d7/pkg/knuu/knuu.go#L204

The app tests should not fail. If they do, can you please point to the failing tests case, so we can have a look and see why it is failing?

Reply:
I can't reproduce the issue anymore. So should be fine if we remove this line 👍